### PR TITLE
Payment 

### PR DESCRIPTION
**Pull request by SophieSjoberg**
### Blockers or problems 
Not yet..

### CODE REVIEW

File: config/initializers/stripe.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Rails.configuration.stripe = {
+#   publishable_key: ENV['PUBLISHABLE_KEY'],
+#   secret_key: ENV['SECRET_KEY']
+# }
+#
+# Stripe.api_key = Rails.configuration.stripe[:secret_key]
```
Comment (tochman) [inline config/initializers/stripe.rb]: Please remove comments

File: config/routes.rb [hunk 2]
```diff
@@ -2,5 +2,7 @@
 
   devise_for :users
   root controller: :landing, action: :index
+  # resources :charges
```
Comment (tochman) [inline config/routes.rb]: Please remove comment

File: features/support/env.rb [hunk 3]
```diff
@@ -15,3 +15,17 @@
 Cucumber::Rails::Database.javascript_strategy = :truncation
 
 World(FactoryBot::Syntax::Methods)
+
+# Chromedriver.set_version '2.33'
```
Comment (tochman) [inline features/support/env.rb]: Please remove comment

**Review by tochman — CHANGES_REQUESTED** [review]

File: features/support/env.rb (hunk 3 above)
Comment (SophieSjoberg) [reply features/support/env.rb]: 👍

File: config/routes.rb (hunk 2 above)
Comment (SophieSjoberg) [reply config/routes.rb]: 👍

File: config/initializers/stripe.rb (hunk 1 above)
Comment (SophieSjoberg) [reply config/initializers/stripe.rb]: 👍

**Pull request closed without merging**